dspace/config/registries/openaire-cerif-types.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0"?>
+
+<dspace-dc-types>
+
+	<dspace-header>
+		<title>OpenAIRE Cris Types</title>
+	</dspace-header>
+
+	<dc-schema>
+		<name>oairecerif</name>
+		<namespace>https://www.openaire.eu/cerif-profile/1.1/</namespace>
+	</dc-schema>
+
+    <dc-type>
+        <schema>oairecerif</schema>
+        <element>identifier</element>
+        <qualifier>url</qualifier>
+        <scope_note></scope_note>
```

Comment:
All fields in this schema require a scope note.  For instance how is this different from dc.identifier.uri?